OpenRA.Mods.Cnc/Traits/Attack/AttackTesla.cs

```diff
@@ -103,7 +110,9 @@ public override Activity Tick(Actor self)
 				if (!string.IsNullOrEmpty(attack.info.ChargeAudio))
 					Game.Sound.Play(SoundType.World, attack.info.ChargeAudio, self.CenterPosition);
 
-				return ActivityUtils.SequenceActivities(self, new Wait(attack.info.InitialChargeDelay), new ChargeFire(attack, target), this);
+				QueueChild(self, new Wait(attack.info.InitialChargeDelay), true);
+				QueueChild(self, new ChargeFire(attack, target), true);
```

Comment:
Setting pretick to true for that second queued activity doesn't make much sense. You already know that it won't tick anyway.

Reply:
Fixed.

Reply:
There is one more instance of this in `Hunt`.

Reply:
Fixed.